database/migrate.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+function missing(dep) {
+	try {
+		require.resolve(dep);
+		return false;
+	} catch (err) {
+		if (err.code !== 'MODULE_NOT_FOUND') throw err;
+		return true;
+	}
+}
+
+function shell(cmd) {
+	child_process.execSync(cmd, {stdio: 'inherit', cwd: __dirname});
+}
+
+if (missing('sqlite')) {
+	shell('npm install sqlite');
+}
+
+const child_process = require('child_process');
+const fs = require('fs');
+const path = require('path');
+const sqlite = require('sqlite');
+
+if (fs.readdirSync(path.resolve(__dirname, './migrations/')).length) {
+	sqlite.open(path.resolve(__dirname, './sqlite.db')).then(function (database) {
```

Comment:
I think we should be able to use `=>` inthis code - we only avoid it in specific top level scripts

Reply:
Oh yeah, I copied this script from my last PR where it was in the build script, so I was trying to make it pre ES6-compliant.